### ISSUE TITLE
Pick the best format for the current user-agent

### ISSUE THREAD

**Issue by evan-buss** [evan-buss/opds-proxy#5]
We currently always prefer the epub format as we can convert that to device specific formats, but if there is a pre-converted format that is suitable for the current user-agent we should just send that rather than converting.

Maybe we should also display all available formats and let the user choose?

**Comment by Zaljerem** [comment]
Yes, please show all available formats. I'm currently trying to set this up for my daughter's brand new Kindle, it seems to be only displaying the kepub converter, not kindlegen, even though it is also in place. The auto-detection is neat but I'd really prefer all available options.
